lib/way_of_working.rb

```diff
@@ -8,3 +8,13 @@
 module WayOfWorking
   class Error < StandardError; end
 end
+
+# Automatically load way_of_working plugins.
+Gem::Specification.
+  select { |gemspec| gemspec.name =~ /\Away_of_working-/ }.
+  collect(&:name).uniq. # multiple versions of a single gem may be installed at once
+  each do |name|
+  require "#{name.gsub('-', '/')}/plugin"
```

Comment:
Does this assume that the only `-` present in the name is the one at the end of `way_of_working-`?

Reply:
No, it just has to start under the WayOfWorking namespace. I plan to move all the current tooling out of the gem. For example, I'll move the Alex.js inclusive language tooling into a new `WayOfWorking::InclusiveLanguage::Alex` gem (ie. `way_of_working-inclusive_language-alex`, so that if someone wanted to implement `WayOfWorking::InclusiveLanguage::SomeOtherTool` they could.

Reply:
It does mean that the plugin.rb file needs to be at the depth of the namespacing of that tool. Perhaps an example gem would be useful?

Reply:
Ah ok, your explanation makes things clear. I don't think I need to see any example gem, unless you have one available straightforwardly?

Reply:
Not at the minute. Knee deep in GitHub hardening, sorry.

Reply:
@ollietulloch to demonstrate this working:

1. create a new project folder and create this `Gemfile`:

```ruby
source 'https://rubygems.org'

gem 'way_of_working', git: 'https://github.com/HealthDataInsight/way_of_working.git', branch: 'feature/v2-wip'
# gem 'way_of_working-inclusive_language-alex', git: 'https://github.com/HealthDataInsight/way_of_working-inclusive_language-alex.git', branch: 'feature/alex'
```
2. `bundle install`
3. `bundle exec way_of_working exec` should not list `inclusive_language` and `bundle exec way_of_working exec inclusive_language` should not work.
4. In the `Gemfile`, comment back in `way_of_working-inclusive_language-alex` plugin gem.
5. `bundle install`
6. `bundle exec way_of_working exec` should list `inclusive_language` and `bundle exec way_of_working exec inclusive_language` should now work.

Reply:
The branch `feature/v2-wip` if a branch based on `feature/plugins` with the inclusive language code removed.